pkg/provider/aws/resources/vpc.go

```diff
@@ -78,6 +79,11 @@ func CreateNetwork(appName string, dag *core.ResourceGraph) *Vpc {
 	dag.AddResource(igw)
 	dag.AddDependency2(igw, vpc)
 
+	CreatePrivateSubnet(appName, "private1", vpc, "10.0.0.0/18", dag)
```

Comment:
Since the endpoints use the subnets, these need to go first now.

Reply:
👍 i had this in my pr as well, but i can wait for you to get this in and update my eks one